### PR TITLE
Task[38146] Build login, register, 404 page

### PR DESCRIPTION
**Pull request by haitt-2557**
**https://edu-redmine.sun-asterisk.vn/issues/38146

### CODE REVIEW

**Comment by haitt-2557** [comment]
ready

File: assets/css/styles/style.css [hunk 1]
```diff
@@ -473,10 +469,13 @@ a {
   -webkit-border-left: 0.08rem solid #e1e1e1;
   -moz-border-left: 0.08rem solid #e1e1e1;
   border-left: 0.08rem solid #e1e1e1; }
+<<<<<<< HEAD
```
Comment (manhnt-0960) [inline assets/css/styles/style.css]: .

**Review by manhnt-0960 — COMMENTED** [review]
self check lại trước khi ready nha e

**Comment by haitt-2557** [comment]
ready

**Pull request merged**